introspection-engine/introspection-engine-tests/tests/commenting_out/mod.rs

```diff
@@ -19,13 +19,30 @@ async fn a_table_without_uniques_should_ignore(api: &TestApi) -> TestResult {
         })
         .await?;
 
-    let dm = if api.sql_family().is_mysql() {
+    let dm = if api.sql_family().is_mysql() && !api.is_mysql8() {
```

Comment:
Why does MySQL 8 need specific handling here? Different default referential actions?

Reply:
This is introspection. Above we create a foreign key without defining any actions, and the database defaults differ.

Reply:
So plain SQL leads to NoAction/NoAction everywhere else, but Restrict/Restrict for MySQL 8 only?

Reply:
Yeah, the endless joys of MySQL...

Reply:
It's always MySQL...